### DISCUSSION TITLE
设置了swagger的api标签的折叠发现没有生效

### DISCUSSION THREAD

**Discussion by KingDracula** [midwayjs/midway#3867] [Q&A]
### Describe the problem（描述问题）

参照了swagger-ui的官方文档，在config中设置了 swagger: {   
      displayOptions: {
        defaultModelsExpandDepth: -1,
        defaultModelExpandDepth: -1,
      }
    },
但是api tags还是全部展开的，没有折叠，请问这个应该怎么设置

### Midway Versions（Midway 版本）

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(not installed)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.13.7)
✓ @midwayjs/bull(not installed)
✓ @midwayjs/bull-board(not installed)
✓ @midwayjs/cache(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.13.7)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(not installed)
✓ @midwayjs/decorator(3.13.7)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(not installed)
✓ @midwayjs/info(3.13.8)
✓ @midwayjs/jwt(not installed)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.13.8)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✖ @midwayjs/redis(current: 3.15.6, allow: ["3.13.7","3.13.8","3.13.9"])
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(3.13.9)
✓ @midwayjs/session(not installed)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(3.13.8)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(not installed)
✖ @midwayjs/upload(current: 3.15.10, allow: ["3.13.7","3.13.8","3.13.9"])
✓ @midwayjs/validate(3.13.8)
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.13.8)
✓ @midwayjs/ws(not installed)

**Comment by czy88840616** [comment]
用自定义的 swagger-initializer.js 方案，因为前端页面的配置，在 node 端实际传不进去。
